ete4/smartview/ete/tree.pyx

```diff
@@ -0,0 +1,210 @@
+"""
+Class to represent trees (which are nodes connected to other nodes) and
+functions to read and write them.
+
+The text representation of the trees are expected to be in the newick format:
+https://en.wikipedia.org/wiki/Newick_format
+"""
+
+from collections import namedtuple
+from ete4 import Tree
+
+TreePos = namedtuple('TreePos', 'node nch')
+# Position on the tree: current node, number of visited children.
+
+
+
+# cdef class Tree:
```

Comment:
Uhm, why do we add the commented-out old version of class Tree?

We don't use it anymore, and its purpose there is unclear. If we want to leave a comment related to some functionality or where to find some references, we could do it with an explanation (with a `# NOTE: ...` for example) and a link?